creator-node/src/components/healthCheck/healthCheckController.js

```diff
@@ -9,24 +9,16 @@ const {
 } = require('../../apiHelpers')
 const {
   healthCheck,
-  healthCheckVerbose,
   healthCheckDuration,
   configCheck
 } = require('./healthCheckComponentService')
 const { syncHealthCheck } = require('./syncHealthCheckComponentService')
 const { serviceRegistry } = require('../../serviceRegistry')
-const { sequelize } = require('../../models')
 const { getMonitors } = require('../../monitors/monitors')
 const TranscodingQueue = require('../../TranscodingQueue')
-
 const { ensureValidSPMiddleware } = require('../../middlewares')
-
 const config = require('../../config')
 
-const router = express.Router()
-
-const numberOfCPUs = os.cpus().length
```

Comment:
moved this into `healthCheckComponentService` since it makes no sense to pass in from here